trino/client.py

```diff
@@ -46,10 +48,13 @@
 from datetime import datetime
 from email.utils import parsedate_to_datetime
 from time import sleep
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import Any, Dict, Iterator, List, Literal, Optional, Tuple, Union, cast
 from zoneinfo import ZoneInfo
 
+import lz4.block
 import requests
+import zstandard
+from typing_extensions import TypedDict
```

Comment:
`TypedDict` was added to the `typing` module in Python 3.8. We can use standard library instead of introducing a new dependency on `typing_extensions`